toolbox/swiftsim/pipeline/start_halo_finding.sh

```diff
@@ -57,6 +57,7 @@ cp ./submission_scripts/submit_particle_splits.sh $HBT_FOLDER
 # Replace the current PWD in those submission scripts, so that they have the global path of the HBT
 sed -i "s@CURRENT_PWD@${PWD}@g" $HBT_FOLDER/submit_HBT.sh
 sed -i "s@CURRENT_PWD@${PWD}@g" $HBT_FOLDER/submit_particle_splits.sh
+sed -i "s@HBT_FOLDER@${HBT_FOLDER}@g" $HBT_FOLDER/submit_particle_splits.sh
```

Comment:
`${HBT_FOLDER}` will be  `/simulation directory/HBTplus/`. Shouldn't it be `${BASE_PATH}` instead when substituting in `submit_particle_splits.sh`, so that it is `/simulation directory/splits/`? The input to `generate_split_file` is the location of SWIFT split files.

Reply:
https://github.com/SWIFTSIM/HBTplus/pull/61/commits/c6faa55f3e6b31933db4e4842276b381981ceaad